zScanner/Views/NewDocument/PhotoSelectorCollectionViewCell.swift

```diff
@@ -84,3 +99,17 @@ class PhotoSelectorCollectionViewCell: UICollectionViewCell {
     }()
 }
 
+extension PhotoSelectorCollectionViewCell: UITextViewDelegate {
+    func textViewDidBeginEditing(_ textView: UITextView) {
+        if textView.textColor == UIColor.lightGray {
+            textView.text = nil
+            textView.textColor = UIColor.black
+       }
+    }
+    func textViewDidEndEditing(_ textView: UITextView) {
+        if textView.text.isEmpty {
+            textView.text = "newDocumentPhotos.description.placeholder".localized
+            textView.textColor = UIColor.lightGray
+        }
+    }
+}
```

Comment:
Use `textField.placeholder`

Reply:
It's TextView, so should I change to TextField?

Reply:
Oh, I didn’t realise. Yeah, try to change it to textField and try out if it works fine. The simpler solution the better. :)